plugins/bean-validation/src/main/java/org/apache/struts/beanvalidation/validation/interceptor/BeanValidationInterceptor.java

```diff
@@ -92,39 +95,43 @@ protected String doIntercept(ActionInvocation invocation) throws Exception {
         ActionProxy actionProxy = invocation.getProxy();
         String methodName = actionProxy.getMethod();
 
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("Validating [{}/{}] with method [{}]", invocation.getProxy().getNamespace(), invocation.getProxy().getActionName(), methodName);
-        }
+        LOG.debug("Validating [{}/{}] with method [{}]", invocation.getProxy().getNamespace(), invocation.getProxy().getActionName(), methodName);
```

Comment:
It seems we must revert deleting `if (LOG.isDebugEnabled()) {`. Please see [René Gielen comment](https://github.com/apache/struts/pull/200#discussion_r161456354)

Reply:
In this case it's safe to leave as is, there is no additional cost to fetch these values - it's a chain of getters.

Reply:
Thank you very much! I approved, however I'm not familiar with bean validation but it covers all codes and not fails. If you're familiar and ok then please merge :)